doc/sphinx/Pacemaker_Administration/pcs-crmsh.rst

```diff
@@ -0,0 +1,441 @@
+
+.. topic:: Remove node "pcmk-1" from standby mode
+
+   .. code-block:: none
+
+      crmsh     # crm node online pcmk-1
+      pcs-0.9   # pcs cluster unstandby pcmk-1
+      pcs-0.10  # pcs node unstandby pcmk-1
+      pacemaker # crm_standby -N pcmk-1 -v off
+
+Manage Cluster Properties
+#########################
+
+.. topic:: Set the "stonith-enabled" cluster property to "false"
+
+   .. code-block:: none
+
+      crmsh     # crm configure property stonith-enabled=false
+      pcs       # pcs property set stonith-enabled=false
+      pacemaker # crm_attribute -n stonith-enabled -v false
+
+Show Resource Agent Information
+###############################
+
+.. topic:: List Resource Agent (RA) Classes
+
+   .. code-block:: none
+
+      crmsh    # crm ra classes
+      pcs      # pcs resource standards
+      pacmaker # crm_resource --list-standards
+
+.. topic:: List Available Resource Agents (RAs) by Standard
+
+   .. code-block:: none
+
+      crmsh     # crm ra list ocf
+      pcs       # pcs resource agents ocf
+      pacemaker # crm_resource --list-agents ocf
+
+.. topic:: List Available Resource Agents (RAs) by OCF Provider
+
+   .. code-block:: none
+
+      crmsh     # crm ra list ocf pacemaker
+      pcs       # pcs resource agents ocf:pacemaker
+      pacemaker # crm_resource --list-agents ocf:pacemaker
+
+.. topic:: List Available Resource Agent Parameters
+
+   .. code-block:: none
+
+      crmsh     # crm ra meta IPaddr2
+      pcs       # pcs resource describe IPaddr2
+      pacemaker # crm_resource --show-metadata ocf:heartbeat:IPaddr2
+
+You can also use the full ``class:provider:type`` format with crmsh and pcs if
+multiple RAs with the same name are available.
+
+.. topic:: Show Available Fence Agent Parameters
+
+   .. code-block:: none
+
+      crmsh # crm ra meta stonith:fence_ipmilan
+      pcs   # pcs stonith describe fence_ipmilan
+
+Manage Resources
+################
+
+.. topic:: Create a Resource
+
+   .. code-block:: none
+
+      crmsh # crm configure primitive ClusterIP ocf:heartbeat:IPaddr2 \
+              params ip=192.168.122.120 cidr_netmask=24 \
+              op monitor interval=30s 
+      pcs   # pcs resource create ClusterIP IPaddr2 ip=192.168.122.120 cidr_netmask=24
+
+pcs determines the standard and provider (``ocf:heartbeat``) automatically
+since ``IPaddr2`` is unique, and automatically creates operations (including
+monitor) based on the agent's meta-data.
+
+.. topic:: Show Configuration of All Resources
+
+   .. code-block:: none
+
+      crmsh    # crm configure show
+      pcs-0.9  # pcs resource show --full
+      pcs-0.10 # pcs resource config
+
+.. topic:: Show Configuration of One Resource
+
+   .. code-block:: none
+
+      crmsh    # crm configure show ClusterIP
+      pcs-0.9  # pcs resource show ClusterIP
+      pcs-0.10 # pcs resource config ClusterIP
+
+.. topic:: Show Configuration of Fencing Resources
+
+   .. code-block:: none
+
+      crmsh    # crm resource show
```

Comment:
Please use `crm resource status` to avoid warning `WARNING: This command 'show' is deprecated, please use 'status'`

Reply:
Thanks!